api/apps/graphql/resolvers.coffee

```diff
@@ -1,11 +1,17 @@
 _ = require 'underscore'
 { where, presentCollection } = Article = require '../articles/model'
+User = require '../users/model.coffee'
+
+module.exports.articles = (root, args, req, ast) ->
+  if (not args.published or args.scheduled) and not args.channel_id
```

Comment:
Ah good thought! I think we could do this on the [Joi side](https://github.com/hapijs/joi/blob/v10.0.6/API.md#alternativeswhenref-options), but I'm not sure if there's a way to bubble up a clear error message like: `Must pass channel_id to view unpublished articles. Or pass published: true to only view published articles.` should someone forget this rule.